### DISCUSSION TITLE
Path autocomplete doesn't work inside `render`, `make`, `view` methods.

### DISCUSSION THREAD

**Discussion by cesarreyes3** [haidubogdan/netbeans-php-blade-plugin#71] [Ideas]
`CTRL + CLICK` works but path autocomplete doesn't work inside `render`, `make`, `view` methods.

**Comment by haidubogdan** [comment]
Hi,
this feature is implemented in another plugin https://github.com/haidubogdan/netbeans-php-laravel-framework, but only for laravel projects.

This plugin more experimental than the blade one :) .

You will need to download the **nbm** associated to your IDE netbeans version from https://github.com/haidubogdan/netbeans-php-laravel-framework/releases/tag/nbmlaravel1.0.4

You might need to disable the go to click action from php blade to avoid concurring actions .

![image](https://github.com/user-attachments/assets/3c041f27-9f0c-490c-b115-1794f43e128e)
